pkgs/top-level/all-packages.nix

```diff
@@ -11302,6 +11302,8 @@ with pkgs;
     pinentry = pinentry-curses;
   };
 
+  ble-serial = with python3Packages; toPythonApplication ble-serial;
```

Comment:
also, I believe the correct way to do this now is just to create an entry in `by-name/bl/ble-serial` with the same line. See https://github.com/NixOS/nixpkgs/pull/434619/files

Reply:
Great, thanks.